### DISCUSSION TITLE
Build Ipa for distribution getting error

### DISCUSSION THREAD

**Discussion by SolidSoftwareDev** [codemagic-ci-cd/codemagic-docs#1400] [Q&A] [answered]
Hi, building ipa for distribution exited with error

If i use the command below, i received status 65
xcode-project build-ipa \
--workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
--scheme "$XCODE_SCHEME"

otherwise, i received status code 9:
xcode-project build-ipa \
--workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
--scheme "$XCODE_SCHEME" \
--archive-flags "-destination 'generic/platform=iOS'" \
--archive-xcargs "CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO" --verbose

RN project.
buidID: 63e94f5c7d70eaba77ac96c6

Thank you.

**Comment by icarusdust** [comment, marked as answer]
Hi @SolidSoftwareDev, keep   `--archive-flags="-destination 'generic/platform=iOS'"` but remove -`-archive-xcargs "CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO"` from the build script and let me know how it goes.

**Reply by SolidSoftwareDev** [reply]
Thank you, it works!!
